src/components/account-avatar.tsx

```diff
@@ -0,0 +1,17 @@
+import { BoxProps, color, DynamicColorCircle } from '@stacks/ui';
+import React from 'react';
+import { Account, getAccountDisplayName } from '@stacks/wallet-sdk';
+
+export const AccountAvatar: React.FC<{ account: Account } & BoxProps> = ({ account, ...props }) => {
+  const displayName = getAccountDisplayName(account);
+  const circleText = displayName?.includes('Account') ? displayName.split(' ')[1] : displayName[0];
+  return (
+    <DynamicColorCircle
+      string={`${account.salt}.${account.dataPrivateKey}::${account.dataPrivateKey}`} // is this an issue?
```

Comment:
Yeah.. lets not, it's just too dicey. Can we just use the public key? Let's not use the `salt`, too.

Ideally we have account gradients that can be the exact same in the explorer, too. Maybe just use the hash part of the address, if we want them to be identical across networks? Although, there is an argument for some differentiation across networks, when displaying on the explorer.

Reply:
Hmm damn this was the best combo that gave the nicest colors. Do you have recommendations how how we can get 3 different strings from the public key, deterministically?